io.github.daid.EmptyEpsilon.metainfo.xml

```diff
@@ -0,0 +1,74 @@
+  
+  <developer id="io.github.daid">
+    <name>daid</name>
+  </developer>
+  
+  <description>
+    <p>EmptyEpsilon is a spaceship bridge simulator game. It&apos;s fully open source, so it can be modified in any way people wish.</p>
+    <p>EmptyEpsilon places you in the roles of a spaceship&apos;s bridge officers, like those seen in Star Trek. While you can play EmptyEpsilon alone or with friends, the best experience involves 6 players working together on each ship.</p>
+    <p>Each officer fills a unique role: Captain, Helms, Weapons, Relay, Science, and Engineering. Except for the Captain, each officer operates part of the ship through a specialized screen. The Captain relies on their trusty crew to report information and follow orders.</p>
+  </description>
+
+  <launchable type="desktop-id">io.github.daid.EmptyEpsilon.desktop</launchable>
+
+  <categories>
+    <category>Game</category>
+    <category>Simulation</category>
+  </categories>
+  
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="social-audio">intense</content_attribute>
+  </content_rating>
+  
+  <url type="homepage">https://daid.github.io/EmptyEpsilon/</url>
+  <url type="bugtracker">https://github.com/daid/EmptyEpsilon/issues</url>
+  <url type="vcs-browser">https://github.com/daid/EmptyEpsilon/</url>
+  
+  <screenshots>
+    <screenshot type="default">
+      <image>https://daid.github.io/EmptyEpsilon/screenshots/mainscreen.png</image>
+      <caption>Main screen view from a cruiser during a large battle.</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://daid.github.io/EmptyEpsilon/screenshots/helms.png</image>
+      <caption>View of the helms officer's screen, which controls the ship's movement.</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://daid.github.io/EmptyEpsilon/screenshots/weapons.png</image>
+      <caption>View of the weapons officer's screen, which controls the ship's weapons and shields.</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://daid.github.io/EmptyEpsilon/screenshots/engineering.png</image>
+      <caption>View of the engineering officer's screen, which can allocate power and repairs to the ship's subsystems.</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://daid.github.io/EmptyEpsilon/screenshots/science.png</image>
+      <caption>View of the science officer's screen, which has long-range scanners and a large database of information.</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://daid.github.io/EmptyEpsilon/screenshots/relay.png</image>
+      <caption>View of the relay officer's screen, which can see and hail allies, share their short-range radar, and launch remote probes.</caption>
+    </screenshot>
+  </screenshots>
+  
+  <releases>
+    <release version="2024.08.09" date="2024-08-09">
```

Comment:
I don't like having hardcoded release numbers and links in the repo.

Reply:
Hmmm, it seems that only version name and date would be enough, maybe that part should be templated somehow. (similar to how [PrismLauncher](https://github.com/PrismLauncher/PrismLauncher/blob/develop/program_info/org.prismlauncher.PrismLauncher.metainfo.xml.in) does it)

Reply:
okay, so I basically copied the approach from PrismLauncher, it is now tested and working, I'll mark this as resolved